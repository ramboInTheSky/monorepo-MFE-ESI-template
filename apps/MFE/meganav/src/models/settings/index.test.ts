 import {SettingsSdkKeys} from "."

 describe("Model - settings: ", () => {
     it("should match the SettingsSdkKeys", () => {
         expect(SettingsSdkKeys).toMatchSnapshot()
     })
 })
