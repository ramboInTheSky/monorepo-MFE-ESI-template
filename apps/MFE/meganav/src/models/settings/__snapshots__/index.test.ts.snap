// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model - settings:  should match the SettingsSdkKeys 1`] = `
Object {
  "CorrelationHeaderName": "x-monorepo-correlation-id",
  "appCache": "meganav.frontend.appCacheTTL",
  "cacheControl": "meganav.frontend.cache-control.max-age",
  "cacheControlFallbackMeganav": "meganav.frontend.cache-control.fallback-meganav.max-age",
  "defaultPrimaryConfig": "meganav.frontend.defaultPrimaryConfig",
  "defaultSecondaryConfig": "meganav.frontend.defaultSecondaryConfig",
  "direction": "meganav.api.direction",
  "imagePlaceHolder": "meganav.frontend.ImagePlaceholder",
  "language": "meganav.api.languageName",
  "languageId": "meganav.alternativeLanguages",
  "realm": "meganav.api.realmName",
  "secondaryMeganavCacheControl": "meganav.frontend.cache-control.secondary-meganav.max-age",
  "template": "meganav.frontend.template",
  "territory": "meganav.api.territoryName",
  "textData": "meganav.frontend.textData",
  "themeVersion": "meganav.frontend.themeVersion",
}
`;
